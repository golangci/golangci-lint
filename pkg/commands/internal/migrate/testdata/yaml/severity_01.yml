issues:
  # Only to not generate unrelated elements inside golden.
  exclude-use-default: false
  # Only to not generate unrelated elements inside golden.
  exclude-generated: strict
  # Only to not generate unrelated elements inside golden.
  exclude-dirs-use-default: false

severity:
  default-severity: error
  case-sensitive: true
  rules:
    - linters:
        - dupl
      severity: info
