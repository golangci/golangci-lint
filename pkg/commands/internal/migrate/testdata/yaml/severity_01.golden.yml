version: "2"
severity:
  default: error
  rules:
    - linters:
        - dupl
      severity: info
